README.md

```diff
@@ -17,8 +13,11 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 The aim of this project is to learn how to use React.
 
-![Screenshot from 2021-08-18 18-16-06](https://user-images.githubusercontent.com/64914462/129924752-757fb6dd-8724-488b-887d-552e63e58f28.png)
+# Project online link
+
+[Click here](https://pasytchangwa.github.io/Math/) to see a sample demo.
```

Comment:
- [ ] Currently, the live link pointing to the README doesn't seem professional, please fix it.

Reply:
Thank you for the review. I will look into it and see how to make it better.

Reply:
fixed!